app/javascript/src/components/dashboard/MyShifts.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
```

Comment:
Please remove all changes you made to this component. Your changes have made some inconsistencies in styling.